test/wpt/text-styles.yaml

```diff
@@ -503,13 +503,34 @@
   testing:
   - 2d.text.measure.boundingBox.textAlign
   code: |
+    ctx.textAlign = "left";
+    leftMetrics = ctx.measureText('hello');
+    assert(leftMetrics.actualBoundingBoxLeft < leftMetrics.actualBoundingBoxRight, "leftMetrics.actualBoundingBoxLeft < leftMetrics.actualBoundingBoxRight");
+
     ctx.textAlign = "right";
-    metrics = ctx.measureText('hello');
-    @assert metrics.actualBoundingBoxLeft > metrics.actualBoundingBoxRight;
+    rightMetrics = ctx.measureText('hello');
+    assert(rightMetrics.actualBoundingBoxLeft > rightMetrics.actualBoundingBoxRight, "metrics.actualBoundingBoxLeft > metrics.actualBoundingBoxRight");
+    
+    ctx.textAlign = "start";
+    
+    ctx.direction = "ltr";
+    ltrStartMetrics = ctx.measureText('hello');
+    assert.deepStrictEqual(ltrStartMetrics, leftMetrics, "ltr start metrics should equal left metrics");
+    	
+    ctx.direction = "rtl";
+    rtlStartMetrics = ctx.measureText('hello');
+    assert.deepStrictEqual(rtlStartMetrics, rightMetrics, "rtl start metrics should equal right metrics");
+
+    ctx.textAlign = "end";
+
+    ctx.direction = "ltr";
+    ltrEndMetrics = ctx.measureText('hello');
+    assert.deepStrictEqual(ltrEndMetrics, rightMetrics, "ltr end metrics should equal right metrics");
+    	
+    ctx.direction = "rtl";
+    rtlEndMetrics = ctx.measureText('hello');
+    assert.deepStrictEqual(rtlEndMetrics, leftMetrics, "rtl end metrics should equal left metrics");
 
-    ctx.textAlign = "left"
-    metrics = ctx.measureText('hello');
-    @assert metrics.actualBoundingBoxLeft < metrics.actualBoundingBoxRight;
```

Comment:
I don't think this goes here. This test is imported from the [WPT tests](https://github.com/web-platform-tests/wpt/blob/ccaadde016187b16e607c6c6ae65e37089c452c5/html/canvas/tools/yaml/text.yaml#L522-L531), so it would get overwritten if they did any updates. You should be able to write this in `canvas.test.js` though.

Reply:
I thought the yaml files generate the js test files. Regardless I saw nothing runs these tests on CI